src/pages/overview/document-generation-api/dynamictableconstructs.md

```diff
@@ -132,3 +132,55 @@ JSON representation of the input data:
 
 - {{Conversion.Rate:**discard-if(expr($sum(Conversion.Rate)!= 100))**}}% tag lets the engine discard the particular column if condition provided in the **expr** construct evaluates to true.
 
+
+## Dynamic table rows
+Discard a row or set of rows in a table from the final generated document.
+
+### Discard row if condition evaluates to true
+Tag: **discard-row-if(expr(condition),context)**
+
+Row in the table can be discarded only when the condition in the discard-row-if(expr(**condition**),**context**) evaluates to true in the provided context. Add this construct along with the template tag to activate discard row if feature for the corresponding rows in the table.
+
+For Instance: {{TABLE_CONTEXT.PROPERTY:**discard-row-if(expr(PROPERTY = "TEST_VALUE"),TABLE_CONTEXT)**}}
```

Comment:
Do not over complicate it by wirting, Table_Context etc things here. Use a simple example to do the same.

Or you can explain what do you mean by context and give one example to explain the content first and later use the same example to explain the actual discard-if-row feature. This is unclear from above.

Reply:
since we are already adding the example below in the picture. 
So a user can easily compare the picture example with this to understand what he should use in expression and what should be used in template tag.

So flow is like introduce the user with 
1. tag first 
2. how to use that tag in creating template tag
3. refer to example given which he can use to test.